src/axolotl/monkeypatch/transformers/tensor_parallel.py

```
@@ -0,0 +1,26 @@
+    model._tp_size = tp_size  # pylint: disable=protected-access
+    model._device_mesh = device_mesh  # pylint: disable=protected-access
+    return res
+
+
+def patch_tp_fix():
+    transformers.integrations.tensor_parallel.distribute_model = distribute_model
+    setattr(
+        sys.modules["transformers.integrations.tensor_parallel"],
+        "distribute_model",
+        distribute_model,
+    )
```

Ruff (0.12.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)